---
title: ""
---

</br> </br>

### Code

Find me on [Github](http://github.com/jtleek)

* [ari](https://github.com/seankross/ari) - An R package for synthesizing voices from text files and R markdown files created by Sean Kross. 
* [matahari](https://github.com/jhudsl/matahari) - An R package for collecting data on the way people analyze data created by Sean Kross. 
* [papr](http://jhubiostatistics.shinyapps.io/papr) - A tindr like web-app for rating preprints from bioRxiv. Written by Lucy D'Agostino McGowan, Nicholas Strayer, and Jeff Leek.
* [Rail-RNA](https://github.com/nellore/rail) Rail-RNA is software for alignment of multiple RNA-seq samples jointly that can be run on a local computer, on a local cluster, or on the AWS cloud. 
* [ballgown](https://github.com/alyssafrazee/ballgown) Statistical backend for analyzing population level data from Cufflinks. Written by Alyssa Frazee, Geo Pertea, and Jeff Leek. 
* [derfinder](https://github.com/lcolladotor/derfinder) Fast differential expression analysis of RNA-seq data at base-pair resolution. Written by Leo Collado Torres, Alyssa Frazee, Andrew Jaffe, Rafa Irizarry, and Jeff Leek.  
* [regionReport](http://master.bioconductor.org/packages/release/bioc/html/regionReport.html) Software for creating interactive, reproducible HTML reports for derfinder. Written by Leo Collado Torres. 
* [polyester](https://github.com/alyssafrazee/polyester) R package for simulating RNA seq reads. Written by Alyssa Frazee and Andrew Jaffe. 
* [sva](http://www.bioconductor.org/packages/devel/bioc/html/sva.html) It has been shown that genome-wide expression may be affected by environmental, demographic, genetic and technical factors, creating what we call expression heterogeneity. Surrogate variable analysis (SVA) is designed to identify, estimate, and incorporate into an analysis the sources of expression heterogeneity that are not captured by variables included in the model. SVA has been shown to reduce dependence across genes, stablize false discovery rate estimates, and improve reproducibility of analyses. Written by Jeff Leek, Evan Johnson, Hilary Parker, Andrew Jaffe and John Storey.
* [validate](http://biostat.jhsph.edu/~jleek/validate/) A key component of performing any genomics experiment is validation of significant features (genes, proteins, etc.). This software can be used to assess the statistical evidence for validation of a particular analysis/technology on the basis of a random sample of significant results. Written by Jeff Leek.
* [tspreg](https://github.com/jtleek/tspreg) Top scoring pairs regression models. For building multiclass and survival top scoring pairs, written by Jeff Leek and Prasad Patil. 
* [dks](http://www.bioconductor.org/packages/2.9/bioc/html/dks.html) The explosive growth of high-dimensional data has resulted in an equally explosive growth in methods for analyzing high-dimensional data. Almost all of these methods rely on p-values, corrected p-values, or false discovery rate estimates for ranking and significance calculation. However, there is no clear standard for determining whether the p-values from a new multiple testing procedure are correct. The double Kolmogorov-Smirnov package consists of a set of R functions for diagnosing whether a multiple testing procedure gives correct null p-values using simulated data.Written by Jeff Leek.
* [myrna](http://bowtie-bio.sourceforge.net/myrna/index.shtml) Myrna is a cloud computing tool for calculating differential gene expression in large RNA-seq datasets. Myrna uses Bowtie for short read alignment and R/Bioconductor for interval calculations, normalization, and statistical testing. These tools are combined in an automatic, parallel pipeline that runs in the cloud (Elastic MapReduce in this case) on a local Hadoop cluster, or on a single computer, exploiting multiple computers and CPUs wherever possible. Written by Ben Langmead, Kasper Hansen, and Jeff Leek.
* [edge](http://bioconductor.org/packages/release/bioc/html/edge.html) A comprehensive software package for the significance analysis of DNA microarray experiments -- for both standard and time course experiments -- based on our new optimal discovery procedure and time course methodology. Written by John Storey, Jeff Leek,and Andrew Bass.
* [tspair](http://www.bioconductor.org/packages/release/bioc/html/tspair.html) A top scoring pair is a pair of genes whose relative ranks can be used to classify arrays according to a binary phenotype. A top scoring pair classifier has three advantages over standard classifiers: (1) the classifier is based on the relative ranks of genes and is more robust to normalization and preprocessing, (2) the classifier is based on a pair of genes and is likely to be more interpretable than a more complicated classifier, and (3) a classfier based on a small number of genes lends itself to diagnostic tests based on PCR that are both more rapid and cheaper than classifiers based on a large number of genes. Written by Jeff Leek.
* [Set](https://github.com/SiminaB/Set) This package implements a decision-theoretic approach to gene-set analysis,
considering a number of different loss functions introduced in the companion Boca et al. (2013) Biometrics paper. Written by Simina Boca, Hector Corrada Bravo, Jeff Leek.

----------

### Data

* [tidypvals](https://github.com/jtleek/tidypvals) - data from over 2.5 million published p-values compiled by Jeff Leek. 
* [recount2](https://jhubiostatistics.shinyapps.io/recount/) - large collection of RNA-seq data created by Leo Collado Torres, Ben Langmead, and Abhi Nellore.  
* [Recount](http://bowtie-bio.sourceforge.net/recount/) a database of free RNA-seq data sets at the gene count level created by Alyssa Frazee
* [Medical Journal P-value Data](https://github.com/jtleek/swfdr/blob/master/pvalueData.rda) - a set of p-values scraped from the abstracts of the New England Journal of Medicine, JAMA, BMJ, The Lancet, and the Amerian Journal of Epidemiology from 2000-2010 created by Jeff Leek. 
* [Processed GEUVADIS data](https://www.dropbox.com/s/kp5th9hgkq8ckom/geuvadisbg.rda) - processed data from 667 RNA-seq samples from the [GEUVADIS](http://www.geuvadis.org/) project created by Alyssa Frazee.
* [GEUVADIS BAM files](http://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-6/) - processed with Tophat2 and created by Alyssa Frazee
* [anitProfiles data](http://bioconductor.org/packages/release/bioc/html/antiProfiles.html) - gene expression data from cancer and normal samples used in our paper on [classifying with variance](http://www.biomedcentral.com/1471-2105/13/272) created by Hector Corrada Bravo.
* [Peer review data](http://www.biostat.jhsph.edu/~jleek/peerreview) - data from our laboratory experiment of peer review created by Jeff Leek and Margaret Taub.
* [Batch effect data](http://rafalab.jhsph.edu/batch/) - data on batch effects from a range of genomic technologies created by Jeff Leek, Rafa Irizarry,Hector Corrada Bravo, Ben Langmead, Keith Baggerly, Evan Johnson, Rob Scharpf, and David Simcha. 
* [Bladder batch data](http://www.bioconductor.org/packages/devel/data/experiment/html/bladderbatch.html) - data on batch effects from a study of bladder cancer in expression set format created by Jeff Leek
* [Braincloud](http://braincloud.jhmi.edu) - data from [our paper](http://www.nature.com/nature/journal/v478/n7370/full/nature10524.html) looking at patterns of variation in gene expression in the human brain created by Carlo Colantuoni. 