<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>
<h2>Code and data for
<i>Tackling the widespread and critical impact of batch effects in high-throughput data</i></h2>


<p>The R scripts included below require various Bioconductor and CRAN packages. Having a <a href="http://www.bioconductor.org/docs/install-howto.html">full installation of Bioconductor</a> will make this much easier. Also CRAN packages <i>corpcor</I> and <i>RColorBrewer</i>.	  


  	  <h3>Example exploratory analysis</h3>
	  <OL>
 	  <LI> This example uses raw data and annotation in <a href="https://ndownloader.figshare.com/files/4862323">this archive</a> [188 MB]</LI>
	  <LI> You need the myplclust function in <a href="myplclust.R">this script</a>.
	  <LI> Then you can run the example in <a href="example.R">this script</a>.
	  </OL>

  	  <h3>Affymetrix Expression Data from Spielman et al.</h3>
	  <OL>
	  <LI> We used processed data as described in the paper "Akey, J.M., Biswas, S., Leek, J.T. & Storey, J.D. On the design and analysis of gene expression studies in human populations. Nat Genet 39, 807-8; author reply 808-9 (2007).". All the data files and scripts needed are in <a href="https://ndownloader.figshare.com/files/4862326">this archive</a>
	  </OL>
	  	  
  	  <h3> Affymetrix Expression Data from Dyrskjot et al.</h3>

	  <OL>
	  <LI> The raw data and annotation is in <a href="https://ndownloader.figshare.com/files/4862323 ">this archive</a> [188 MB]</LI>
	  <LI> Use the <a href="bladder.R">bladder.R</a> script to preprocess data. This will produce an rda file needed in the next step.</LI>
	  <LI> Run the <a href="figure1.R">figure1.R</a> script to produce Figure 1.
	  <LI> Run the <a href="bladder-analysis.R">bladder-analysis.R</a> script to obtain results in table.
	  </OL> 

	<h3> Mass Spec Data</h3>
	  <OL>
	  <LI> <a href="buildProt.pdf">This document</a> explains the analysis performed here in detail. </LI>
	  <LI> <a href="buildProt.Rnw">This Sweave file</a> was used to produce the pdf above.
	  <LI> You can download the rather large <a href="http://bioinformatics.mdanderson.org/Supplements/BatchPaper/allProtein.Rda">R object</a> that it builds.
	  <LI> Finally, to produce results seen Table 1 you run <a href="mass_spec.R">this script</a>. You will need <a href="https://ndownloader.figshare.com/files/4862335">these csv files</a> which are produced by the Sweave document above.

	  </OL> 

	  <h3> Copy Number Data</h3>
	  <OL>
	  <LI> You can obtain GWAS data from <a href="http://www.ncbi.nlm.nih.gov/gap">dbGap</a>... but it is not easy. We can't post it here.
	  <LI> The HapMap data is available from <a href="http://hapmap.ncbi.nlm.nih.gov">HapMap</a>.
	  <LI> <a href="https://ndownloader.figshare.com/files/4862329">This R package</a> contains the code necessary to produce results in Table 1. You will find two vignettes in inst/doc: <i>snpAnalysis.Rnw</I> and  <i>bip.Rnw</i>. <i>snpAnalysis.Rnw</i> is the analysis of the HapMap phase 3 data.  <i>bip.Rnw</i> is the analysis of the european ancestry bipolar data. 
	  </OL>


<h3>TCGA Affymetrix Expression Data</h3>
	  <OL>
<!--	  <LI> Download <a href="ftp://ftp1.nci.nih.gov/tcga/tumor/ov/cgcc/broad.mit.edu/ht_hg-u133a/transcriptome/">raw data</a>.</LI> -->
 <LI> Download <a href="https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/ov/cgcc/broad.mit.edu/ht_hg-u133a/transcriptome/">raw data</a>.</LI>
	  <LI> Use the <a href="join-affy.R">join-affy.R</a> script to preprocess data. You will have to edit script so that correct path for raw data is used. This will produce an rda file needed in the next step.</LI>
	  <LI> Run the <a href="tcgaaffy.R">tcgaaffy.R</a> script to obtain results in table.
	  </OL>
	  


  	  <h3>TCGA Agilent Expression Data</h3>
	  <OL>
<!--	  <LI> Download <a href="ftp://ftp1.nci.nih.gov/tcga/tumor/ov/cgcc/unc.edu/agilentg4502a_07_3/transcriptome/">raw data</a>.</LI>-->
  <LI> Download <a href="https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/ov/cgcc/unc.edu/agilentg4502a_07_3/transcriptome/">raw data</a>.</LI>
	  <LI> Use the <a href="join-agilent.R">join-agilent.R</a> script to preprocess data. You will have to edit script so that correct path for raw data is used. This will produce an rda file needed in the next step.</LI>
	  <LI> Run the <a href="tcgaagilent.R">tcgaagilent.R</a> script to obtain results in table.
	  </OL>
	
	<h3>TCGA Illumina Methylation Data</h3>
	  <OL>
  	   <LI> Download <a href="https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/ov/cgcc/jhu-usc.edu/humanmethylation27/methylation/">raw data</a>.</LI>

	<!--  <LI> Download <a href="ftp://ftp1.nci.nih.gov/tcga/tumor/ov/cgcc/jhu-usc.edu/humanmethylation27/methylation/">raw data</a>.</LI>-->
	  <LI> Use the <a href="join-meth.R">join-meth.R</a> script to preprocess data. You will have to edit script so that correct path for raw data is used. This will produce an rda file needed in the next step.</LI>
	  <LI> Run the <a href="tcgamethylation.R">tcgamethylation.R</a> script to obtain resultsin table.
	  </OL>
	  
	<h3>Sec-gen sequencing data</h3>
	  <OL>
	  <LI> <a href="https://ndownloader.figshare.com/files/4862332">This archive</a> contains the processed data, annotation, and scripts to produce Figure 2 and table results for sequencing.
	  <LI> To processed data is coverage and was computed from the raw data available from the amazon cluster. This was a computationally intensive job and we ran it on the amazon cluster. The archive above has manageable tables produced from these computations. Specific instructions on how we computed coverage follow:
The table files are constructed by obtaining the .bam files for all of the relevant HapMap individuals, parsing the alignment coordinates for all aligned reads using SAMtools and perl, placing each read into a non-overlapping 1k genomic bin according to the "leftmost" position covered with respect to the forward reference strand, then tallying count totals for each bin and combining the totals into a table.  In the table each successive row is a successive bin and each column is a distinct HapMap sample.


The Myrna command used (the paths are specific to our directory structure -- other users must insert their own):

<PRE>
perl myrna.pl \
   -input s3n://myrna-emr/example/1kgenome/batch_effect/preproc \
   -output s3n://ec2-jeff-tmp/example/1kgenome/batch_effect2/output \
   -count s3n://ec2-jeff-tmp/example/1kgenome/batch_effect2/output/count \
   -ditch-alignments \
   -nodes $NODES \
   -wait-on-fail \
   -sam-passthrough \
   -sam-lab-rg \
   -influence 1 \
   -from5prime \
   -bin 1000 \
   -bypass-pvals \
   $*
</PRE>

The relevant output is in the path specified with the -count option. </LI>

</OL>


 <h3>Change of correlation structure</h3>
Finally, this <a href="signReversal.py">python code</a> used to produce the supplemental plot related to genes that change relative rank across batch. These are used to produce Figure 3. 
	  
	  

    
  </body>
  
</html>
